packages/web-components/fast-foundation/src/tree-view/tree-view.ts

```diff
@@ -88,24 +111,89 @@ export class TreeView extends FoundationElement {
             case keyHome:
                 if (this.treeItems && this.treeItems.length) {
                     TreeItem.focusItem(this.treeItems[0]);
+                    this.treeItems[0].setAttribute("tabindex", "0");
                 }
                 break;
             case keyEnd:
                 if (this.treeItems && this.treeItems.length) {
                     TreeItem.focusItem(this.treeItems[this.treeItems.length - 1]);
+                    this.treeItems[this.treeItems.length - 1].setAttribute(
+                        "tabindex",
+                        "0"
+                    );
+                }
+                break;
+            case keyArrowLeft:
+                e.preventDefault();
```

Comment:
You don't need the preventDefaults here as the template binding does that by default.